src/js/tracks/video-track-enums.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @file video-track-enums.js
+ */
+
+/**
+ * https://html.spec.whatwg.org/multipage/embedded-content.html#dom-videotrack-kind
+ *
+ * enum VideoTrackKind {
+ *   "alternative",
+ *   "captions",
+ *   "main",
+ *   "sign",
+ *   "subtitles",
+ *   "commentary",
+ *   "",
+ * };
+ */
+const VideoTrackKind = {
```

Comment:
Audio and Video Tracks share a lots of enums and are listed in the same list. Many of them are audio only and many of them our video only.